chainio/clients/builder.go

```diff
@@ -45,7 +44,7 @@ type Clients struct {
 	PrometheusRegistry     *prometheus.Registry  // Used if avs teams need to register avs-specific metrics
```

Comment:
should we make this ETH clients private in these `Clients` struct? We should only expose the clients which we want them to use

Reply:
They might need it to call json-rpc methods like blockNumber. This was your insight remember? :P

Reply:
hmm. I guess this is just a mixture of clients so it's fine.